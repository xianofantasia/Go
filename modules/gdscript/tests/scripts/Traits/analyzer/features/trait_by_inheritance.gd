extends OtherClass

class OtherClass extends SomeClass:
	uses SomeTrait

class SomeClass:
	uses OtherTrait

trait SomeTrait:
	pass

trait OtherTrait:
	pass

func test():
	print(self is SomeTrait)
	print(self is OtherTrait)
	print("ok")
