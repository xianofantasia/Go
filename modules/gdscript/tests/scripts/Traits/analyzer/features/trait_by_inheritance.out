GDTEST_OK
true
true
ok
